deps/linenoise/linenoise.c

```diff
@@ -181,6 +185,7 @@ enum KEY_ACTION{
 	CTRL_N = 14,        /* Ctrl-n */
 	CTRL_P = 16,        /* Ctrl-p */
 	CTRL_R = 18,        /* Ctrl-r */
+	CTRL_S = 19,        /* Ctrl-r */
```

Comment:
The comment is wrong

Reply:
hey @zuiderkwast , I just pushed a fix for this, sorry I overlooked it